a = "a"
try:
    a = int(a)
except ValueError as e:
    print('a')